plugins/scaffolder/package.json

```diff
@@ -58,6 +58,7 @@
     "@backstage/plugin-catalog-react": "workspace:^",
     "@backstage/plugin-permission-react": "workspace:^",
     "@backstage/plugin-scaffolder-common": "workspace:^",
+    "@backstage/plugin-scaffolder-node": "workspace:^",
```

Comment:
You can actually not do this. A frontend package shouldn't import a backend (Node) one.

If you need access to these in the frontend too, you may want to migrate them to `@backstage/plugin-scaffolder-common` and deprecate them in `-node`. But I'm not sure if @benjdlambert has opinions about moving them there.